### PR TITLE
added parameter support for url

### PR DESCRIPTION
**Pull request by warcops**
Github issue: #37 

Changes proposed in this PR:
- Added support for url being passed as parameter.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -48,11 +48,16 @@ function hideLoader() {
   loader.style.display = 'none';
 }
 
+getOpenEuroUrl = () => {
```
Comment (Irev-Dev) [inline script.js]: I'm getting console error that reads in firefox `ReferenceError: assignment to undeclared variable getOpenEuroUrl` which makes sense to be as this line should probably be `const getOpenEuroUrl = () => {` 

Though I'd prefer if you stuck to the convention in the rest of the file and use `function getOpenEuroUrl`.

I also don't understand the name of this function, is it supposed to be `getOpenNeuroUrl`?

File: script.js [hunk 2]
```diff
@@ -48,11 +48,16 @@ function hideLoader() {
...
+  const { location } = window;
+  return location.href.substring(location.href.indexOf(location.search) + 1)
```
Comment (Irev-Dev) [inline script.js]: I prefer it if this still fell back to the old hard coded url, perhaps
``` javascript
  const fallBackUrl = 'https://openneuro.org/crn/datasets/ds001417/files/sub-study002:ses-after:anat:sub-study002_ses-after_T1w.nii.gz';
  const { location } = window;
  const passedInUrl = location.href.substring(location.href.indexOf(location.search) + 1)

  return passedInUrl || fallBackUrl;
```

**Comment by warcops** [comment]
the tests failed for you because of me using arrow function instead of the conventional one. I agree with the fallback approach. Have pushed changes. Would request your review. Thanks.

**Comment by Irev-Dev** [comment]
Sorry I feel like I'm waisting your time, I have netlify set up to build deploy previews for pull requests, so we can view yours at 
https://deploy-preview-38--mrislice.netlify.com/
(with error `Unhandled promise rejection TypeError: "NetworkError when attempting to fetch resource."`)
But it doesn't work, and when I put a link to a data set it doesn't work either.
https://deploy-preview-38--mrislice.netlify.com/?https://openneuro.org/crn/datasets/ds001545/snapshots/1.0.0/files/sub-01:anat:sub-01_T1w.json
(with error `Unhandled promise rejection incorrect header check`)
😢
Thanks for the effort.

**Pull request closed without merging**